docs/reference-architectures/virtual-machines-linux/n-tier.md

```diff
@@ -23,7 +23,7 @@ This reference architecture shows a set of proven practices for running Linux vi
 
 There are many ways to implement an N-tier architecture. The diagram shows a typical 3-tier web application. This architecture builds on [Run load-balanced VMs for scalability and availability][multi-vm]. The web and business tiers use load-balanced VMs.
 
-* **Availability sets.** Create an [availability set][azure-availability-sets] for each tier, and provision at least two VMs in each tier.  This makes the VMs eligible for a higher [service level agreement (SLA)][vm-sla] for VMs.
+* **Availability sets.** Create an [availability set][azure-availability-sets] for each tier, and provision at least two VMs in each tier.  This makes the VMs eligible for a higher [service level agreement (SLA)][vm-sla] for VMs. You can deploy a single VM in an availability set, but the single VM will not qualify for a SLA guarantee unless the single VM is using Azure Premium Storage.
```

Comment:
should it be "an SLA"? 

To be very explicit do you want to add "... for all OS and data disks"? (per the SLA doc)